### ISSUE TITLE
Default setenv.sh in $CATALINA_HOME ignored if there is setenv.sh in the $CATALINA_BASE

### ISSUE THREAD

**Issue by nike38rus** [tomcat-slf4j-logback/tomcat-slf4j-logback#154]
Hello!

I'm using single Tomcat installation for launching multiple Tomcat instances on the same server. 
And I want to totally disable JULI so I extraced release archive in the $CATALINA_HOME and removed $CATALINA_HOME/conf/logging.properties.
I'm using setenv.sh to define CLASSPATH variable for some instances. And if there is local one $CATALINA_BASE/bin/setenv.sh, global one $CATALINA_HOME/bin/setenv.sh will not be included in the startup script:
```
if [ -r "$CATALINA_BASE/bin/setenv.sh" ]; then
  . "$CATALINA_BASE/bin/setenv.sh"
elif [ -r "$CATALINA_HOME/bin/setenv.sh" ]; then
  . "$CATALINA_HOME/bin/setenv.sh"
fi
```

My suggestion is to remove $CATALINA_HOME/bin/setenv.sh and add block in the  $CATALINA_HOME/bin/catalina.sh startup script like this:
```
if [ -r "$CATALINA_BASE/conf/logback.xml" ]; then
  JAVA_OPTS="-Djuli-logback.configurationFile=$CATALINA_BASE/conf/logback.xml $JAVA_OPTS"
elif [ -r "$CATALINA_HOME/conf/logback.xml" ]; then
  JAVA_OPTS="-Djuli-logback.configurationFile=$CATALINA_HOME/conf/logback.xml $JAVA_OPTS"
fi
```
and place it before setenv.sh handling to keep ability to redefine `juli-logback.configurationFile` in the $CATALINA_BASE/bin/setenv.sh

This will let us to use custom logback.xml located in the $CATALINA_BASE/conf or use default in the $CATALINA_HOME/conf

**Comment by hazendaz** [comment]
@nike38rus setenv.sh is a script file so it should be able to run that script.  The idea behind using that is to not mess with tomcat confirmation files as tomcat generally suggests in such a case.  If you make that change in setenv.sh does it not work as you expect?

**Comment by nike38rus** [comment]
@hazendaz sure it's working if I put this option in $CATALINA_BASE/bin/setenv.sh, but my point is if somebody wants to totally disable JULI by extracting release archive into $CATALINA_HOME and deleting $CATALINA_HOME/conf/logging.properties, as described in README, it will working only if there is no $CATALINA_BASE/bin/setenv.sh script or if manually add Java option in it. 
So placing Java option into $CATALINA_HOME/bin/setenv.sh is not working in case there is local one $CATALINA_BASE/bin/setenv.sh without `juli-logback.configurationFile` defined in it. It can be workarounded by adding block from comment above in the $CATALINA_HOME/bin/catalina.sh

Anyway, I guess this case should be mentioned in README

**Comment by hazendaz** [comment]
@nike38rus Can you write up the use case and add it to the README?  I'll merge that.

**Comment by hazendaz** [comment]
closed via #162
